content/drone-plugins/drone-gcf/index.md

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
It should be placed in content/oliver06/drone-gcf

Reply:
done